browser_use/agent/message_manager/service.py

```diff
@@ -64,7 +57,9 @@ def __init__(
 			context_message = HumanMessage(content='Context for the task' + self.message_context)
 			self._add_message_with_tokens(context_message)
 
-		task_message = self.task_instructions(task)
+		task_message = HumanMessage(
+			content=f'Your ultimate task is: """{task}""". If you achieved your ultimate task, stop everything and use the done action in the next step to complete the task. If not, continue as usual.'
+		)
```

Comment:
i think we should keep it as a function so that we can call it from outside for new tasks